ui/src/app/join-lobby-link/join-lobby-link.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core';
+import { ApiService } from '../_services/api.service';
+import { ToastrService } from 'ngx-toastr';
+import { RedirectionService } from '../_services/redirection.service';
+import { ActivatedRoute } from '@angular/router';
+import { FormBuilder, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-join-lobby-link',
+  templateUrl: './join-lobby-link.component.html',
+  styleUrls: ['./join-lobby-link.component.scss']
+})
+export class JoinLobbyLinkComponent implements OnInit {
+
+  lobbyId;
+
+  joinLobbyForm = this.fb.group({ userName: ['', Validators.required ],});
+
+  constructor(
+    private api: ApiService,
+    private toaster: ToastrService,
+    private redirectionService: RedirectionService,
+    private route: ActivatedRoute,
+    private fb: FormBuilder,
+  ) { }
+
+  ngOnInit() {
+  }
```

Comment:
If you don't use it remove it. And also remove  `implements OnInit` as it won't be true anymore.